cfgov/jinja2/v1/_includes/molecules/social-media.html

```diff
@@ -0,0 +1,147 @@
+
+   ========================================================================== #}
+
+{# TODO: Remove global page reference when moving to Wagtail. #}
+{% macro render(is_share_view=true, options={}) %}
+{% set page = page or global_dict.page %}
+{% set title = options.title | urlencode or page.seo_title | urlencode %}
+{% set parsed_url = request.url | urlencode %}
+<div class="m-social-media">
+    {% if is_share_view %}
+        <p class="h5 m-social-media_heading">Share this</p>
+    {% endif %}
+
+    <ul class="list
+               list__unstyled
+               list__horizontal
+               m-social-media_icons">
+
+        {% set facebook_info = {
+            'name': 'Facebook',
+            'homepage': 'https://facebook.com/cfpb',
+            'share_url': 'https://www.facebook.com/dialog/share?app_id=210516218981921&amp;display=page&amp;href=' ~ parsed_url ~ '&amp;redirect_uri=' ~ parsed_url,
+            'class': 'cf-icon-facebook-square'
+        } %}
+
+        {% set twitter_info = {
+            'name': 'Twitter',
+            'homepage': 'https://twitter.com/cfpb',
+            'share_url': _share_twitter_url(parsed_url, title, options),
+            'class': 'cf-icon-twitter-square'
+        } %}
+
+        {% set linkedin_info = {
+            'name': 'LinkedIn',
+            'homepage': 'https://www.linkedin.com/company/consumer-financial-protection-bureau',
+            'share_url': 'https://www.linkedin.com/shareArticle?mini=true&url=' ~ parsed_url ~ '&title=' ~ title ~ '%20Developer%20Network',
+            'class': 'cf-icon-linkedin-square'
+        } %}
+
+        {% set email_info = {
+            'name': 'email',
+            'share_url': 'mailto:?subject=' ~ title ~ '&amp;body=Check out this page from the CFPB - ' ~ parsed_url,
+            'class': 'cf-icon-email-social-square'
+        } %}
+
+        {% set youtube_info = {
+            'name': 'YouTube',
+            'homepage': 'https://www.youtube.com/user/cfpbvideo',
+            'class': 'cf-icon-youtube-square'
+        } %}
+
+        {% set flickr_info = {
+            'name': 'Flickr',
+            'homepage': 'https://www.flickr.com/photos/cfpbphotos',
+            'class': 'cf-icon-flickr-square'
+        } %}
+
+        {% if is_share_view %}
+            {% set links = [
+                facebook_info,
+                twitter_info,
+                linkedin_info,
+                email_info
+            ] %}
+        {% else %}
+            {% set links = [
+                facebook_info,
+                twitter_info,
+                linkedin_info,
+                youtube_info,
+                flickr_info
+            ] %}
+        {% endif %}
+
+
+        {% for link in links %}
+            {% if is_share_view %}
+                <li class="list_item">
+                    <a class="m-social-media_icon"
+                       href="{{ link['share_url'] }}"
+                       target="_blank">
```

Comment:
Can we condition this out for the mailto link so that the user isn't left with a blank page after returning from their mail client?